File: ccontreras11.java
// Christian A. Contreras
// LAB11
// 11/14/23

import java.util.Scanner;

public class ccontreras11
{
	public static void main(String[] args)
	{
		Scanner keyboard = new Scanner(System.in);

		System.out.print("Enter a number: ");
		int userInput = keyboard.nextInt();

		System.out.printf("Your number squared is %d\n", userInput*userInput);
	}
}

File: ccontreras11.c
// Christian A. Contreras
// CSC 321-40
// LAB11 (11/14/23)

#include <stdio.h>
#include <stdlib.h>

int main(void)
{
	int userInput;

	printf("Enter a number: ");
	scanf("%d", &userInput);

	printf("Your number squared is %d\n", userInput*userInput);

	return EXIT_SUCCESS;
}
